src/callbackListener.ts

```diff
@@ -15,7 +16,11 @@ export class CallbackListener implements PackageListener {
       this.authUser,
       this.authPassword
     );
-    const response = await fetch(`${this.url}/packagerCallback/success`, {
+    const fetchUrl = new URL(
+      PathUtils.join(this.url.pathname, 'packagerCallback/success'),
```

Comment:
I think this can actually be simplified to `new URL("/packagerCallback/success", this.url)`

Reply:
Ah, yes you are right. I was not aware that `this.url` contained a path (which is of course can do)